vm/src/execute.rs

```diff
@@ -93,6 +107,12 @@ pub fn execute(
     // Deploy if this is a deployment transaction and spend part is successful.
     contract_deploy(session, tx, config, &mut receipt);
 
+    if let Some(blob_min_charge) = blob_min_charge {
+        if receipt.gas_spent < blob_min_charge {
+            receipt.gas_spent = blob_min_charge;
+        }
+    }
```

Comment:
Can you add a comment above this block explaining the rationale of overriding the gas-spent? e.g. explaining the different gas handling as opposed to a deployment transaction